### PR TITLE
i18n: setup + forecast translation

### PR DESCRIPTION
**Pull request by karlguillotte**
Currently, two buttons appear in the footer to switch between locale. Not ideal, it should be better in the navigation bar. Looking suggestions ;) 

- [x] Make sure danger rating mountain texts renders correctly
- [x] Danger ratings in table does not react to locale changes
- [x] Better UI for the locale switch
- [x] Make sure we polyfill all Intl APIs being used
- [x] Make sure we are not missing any translations
- [x] Export bunch of messages from the code, I would like to start with the MIN!

### CODE REVIEW

File: client/layouts/ast/constants.js [hunk 1]
```diff
@@ -7,11 +9,56 @@ export const LEVELS = new Map([
...
+        ['SNOWSHOE', intl.formatMessage({
+            defaultMessage: 'SNOWSHOE',
+            description: 'Constants layouts/ast',
+        })],
+        ['YOUTH', intl.formatMessage({
+            defaultMessage: 'YOUTH',
+            description: 'Constants layouts/ast',
+        })]
+    ]))
+}
```
Comment (karlguillotte) [inline client/layouts/ast/constants.js:62]: Also, I think we should capitalize these. Like in AvySavvy, we should do that in CSS.

**Review by karlguillotte — COMMENTED** [review]
I have extracted all messages and I have noticed few things. See below for more details.

File: client/constants/sortings.js [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+    [DESC, intl.formatMessage({
+      defaultMessage: 'desc',
+      description: 'Constants prismic',
+    })],
+    [NONE, intl.formatMessage({
+      defaultMessage: 'none',
+      description: 'Constants prismic',
+    })]
+  ]))
+}
```
[the remark this answers is not in the document]
Comment (rmcwhae) [reply client/constants/sortings.js]: Removed!

**Review by karlguillotte — COMMENTED** [review]
Thanks!

File: client/layouts/ast/constants.js (hunk 1 above)
Comment (rmcwhae) [reply client/layouts/ast/constants.js]: Updated with Capital case. I skipped adding `text-transform: uppercase` in CSS as the other form controls on the Courses/Providers pages are all Capital case, though this is not consistent elsewhere on the site.

**Pull request closed without merging**